### ISSUE TITLE
Support for chef 13

### ISSUE THREAD

**Issue by Ansonator** [cerner/cerner_splunk#156]
The metadata.rb from this cookbook is refusing to accept newer versions of chef client.  We might be able to work around this in the short term by downgrading chef-client on a lot of servers, but that might lead to other issues.

``` 
ERROR: Cookbook 'cerner_splunk' version '2.14.0' depends on chef version ["~> 12.4"], but the running chef version is 13.5.3
```

**Comment by thisiskushal** [comment]
Millennium Splunk team worked with CWx Tech Improvement (TI) to embed clientops_splunk cookbook into the millApp role to install/update Splunk coverage on an automated basis every 24 hours. The clientops_splunk cookbook is a wrapper for cerner_splunk cookbook and is currently constrained by the chef version limitation. TI has removed the cookbook from the role due to this and that's affecting our ability to auto-onboard client nodes which was a huge cost saving. 

I understand that supporting a new version takes time and effort but I would like to know the plans around this change and tentative timelines as well.

cc: @acharlieh @withrow

**Comment by acharlieh** [comment]
Unfortunately I and the others on the primary maintaining team doesn't currently have capacity to tackle this change ourselves at the moment. As this is an open source project. You are more than welcome to test and contribute. Keep in mind with such a change, we'll likely need to maintain compatibility with later Chef 12 releases, to make the transition as easy as possible. We'd likely want to go to Chef 14 Support for the longest lifetime possible. (In certain ways it's too bad that Chef doesn't have a LTS concept like Ubuntu and other packages).

There is also an issue ( #142 ) that you will likely run into with upgrading Splunk that will need to be dealt with as well as when this cookbook is updated.

**Comment by thisiskushal** [comment]
I have forked this repo and am looking to test this cookbook without the chef version dependency. Will update here when I am ready to create a PR.

**Comment by gravesb** [comment]
Fixed by #174.
